### PR TITLE
Create SONGLYRICS.com.xml

### PR DESCRIPTION
**Pull request by christianheinrichs**

### CODE REVIEW

File: src/chrome/content/rules/SONGLYRICS.com.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<!--
+	Bad gateway:
+		- m.songlyrics.com
+
+	MCB:
+		- http://l.yimg.com/pb/webplayer/0.9.30/js/player-min.js
+
+	Server not found:
```
Comment (J0WI) [inline src/chrome/content/rules/SONGLYRICS.com.xml]: There are no DNS records for them, so you can remove the comments.

File: src/chrome/content/rules/SONGLYRICS.com.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<!--
+	Bad gateway:
+		- m.songlyrics.com
+
+	MCB:
+		- http://l.yimg.com/pb/webplayer/0.9.30/js/player-min.js
```
Comment (J0WI) [inline src/chrome/content/rules/SONGLYRICS.com.xml]: Does this break any functionality?

File: src/chrome/content/rules/SONGLYRICS.com.xml (hunk 1 above)
Comment (christianheinrichs) [reply src/chrome/content/rules/SONGLYRICS.com.xml]: Done.

Comment (J0WI) [reply src/chrome/content/rules/SONGLYRICS.com.xml]: Thanks!

**Pull request merged**